### ISSUE TITLE
Markdown utils are broken in v2.9

### ISSUE THREAD

**Issue by YaraslauZhylko** [aiogram/aiogram#350]
## Context

Markdown utils seem to be broken in 2.9.

Raise `AttributeError: 'function' object has no attribute 'format'` instead of returning the decorated text.

* Operating System: Ubuntu 18.04.4 LTS
* Python Version: 3.8.0
* aiogram version: 2.9
* aiohttp version: 3.6.2.
* uvloop version (if installed): 0.14.0

## Expected Behavior

Markdown utility fucntions like `bold`, `hbold`, `italic`, `italics`, etc. are supposed to return decorated text.
```python
>>> aiogram.utils.markdown.hbold('test')
'<b>test</b>
```

## Current Behavior

Calling markdown utils raises an exception:
```python
AttributeError: 'function' object has no attribute 'format'`
```
This happens to a number of markdown methods: `bold`, `hbold`, `italics`, `hitalics`, etc.

## Failure Information

```python
>>> aiogram.utils.markdown.hbold('test')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/.../.virtualenvs/.../lib/python3.8/site-packages/aiogram/utils/markdown.py", line 85, in hbold
    return html_decoration.bold.format(
AttributeError: 'function' object has no attribute 'format'
```

### Steps to Reproduce

1. Import markdown utility function: `from aiogram.utils.markdown import hbold`
2. Call the function with any parameter: `hbold('test')`
3. Get the exception.

### Failure Logs

More detailed pytest log:
```python
sep = ' ', content = ('John',)

    def hbold(*content, sep=" "):
        """
        Make bold text (HTML)

        :param content:
        :param sep:
        :return:
        """
>       return html_decoration.bold.format(
            value=html_decoration.quote(_join(*content, sep=sep))
        )
E       AttributeError: 'function' object has no attribute 'format'

../../../.virtualenvs/.../lib/python3.8/site-packages/aiogram/utils/markdown.py:85: AttributeError
```

**Comment by YaraslauZhylko** [comment]
Updated from v2.8 to v2.9 and ran the tests. :sweat_smile:

**Comment by YaraslauZhylko** [comment]
Something has probably gone wrong with backporting:
https://github.com/aiogram/aiogram/commit/d5169a294f03c63e62589de2566725b8b7fcc08d

**Comment by theaspirational** [comment]
Same issue

**Comment by Birdi7** [comment]
Thank you for great issue description! It was a pleasure to fix the bug. The issue will be fixed in the next version of the framework. 
Fix in #353
